requirements.txt

```diff
@@ -1,30 +1,21 @@
 appdirs==1.4.0
-beautifulsoup4==4.4.1
-cffi==1.6.0
-cryptography==1.4
-decorator==4.0.9
-enum34==1.1.6
-funcsigs==1.0.2
+beautifulsoup4==4.5.1
+decorator==4.0.10
 future==0.15.2
-gpsoauth==0.3.0
+gpsoauth==0.4.0
 httplib2==0.9.2
-idna==2.1
-ipaddress==1.0.16
 MechanicalSoup==0.4.0
 mock==2.0.0
-mutagen==1.34
-ndg-httpsclient==0.4.0
-oauth2client==2.1.0
+mutagen==1.34.1
+oauth2client==3.0.0
```

Comment:
We'll have to keep an eye on this one. I was recently burned by an oauth2client upgrade on a different project.